shared/checks/oval/installed_app_is_ocp4.xml

```diff
@@ -1,28 +0,0 @@
-<def-group>
```

Comment:
Instead of being removed, this needs to be updated to handle coreos as well as rhel nodes.